metabase/metabase
# Rearrange buttons in the dashboard editing panel

Product doc https://www.notion.so/metabase/Rearrange-buttons-in-the-dashboard-editing-panel-8432eb8df6e449c1ab69149ced425978

Changes:
- Moves `temporal-unit` parameters to the same popover where other parameters are
- Copy & layout changes

How to verify:
- Edit dashboard -> Click filter button
- Make sure that each parameter button is operational

<img width="413" alt="Screenshot 2024-08-20 at 14 01 10" src="https://github.com/user-attachments/assets/2eb13178-9abc-4542-8eb6-56d13d037f0a">
